molecule/pki/verify.yml

```diff
@@ -44,8 +44,39 @@
     - name: Populate services facts
       ansible.builtin.service_facts:
 
+    - name: Create log files based on journalctl
+      when: ansible_os_family == 'RedHat'
+      ansible.builtin.shell:
+        cmd: journalctl -u {{ item }} > /var/log/alfresco/{{ item }}.log
+      loop:
+        - alfresco-tengine-aio
+        - alfresco-transform-router
+        - alfresco-shared-fs
+      become: true
+
+    - name: Check in logs a client did connect
```

Comment:
use set_facts to populate a var depending on the os family to be feed to the `loop` structure so you don't duplicate the entire slurp task